docs/VerifPlans/Simulation/privileged_spec/CV32E40X_Counters.json

```diff
@@ -376,19 +376,19 @@
         "Feature": "Debug Mode",
         "Sub Feature": "",
         "Feature Description": "",
-        "Verification Goal": "hpmcounter[31-3] continues during debug",
+        "Verification Goal": "hpmcounter[31-3] continues during debug if stopcount=0, and stop if stopcount=1",
         "Pass/Fail Criteria": "Self Checking Test",
         "Test Type": "Directed Self-Checking",
         "Coverage  Method": "Testcase",
-        "Link to Coverage": "",
+        "Link to Coverage": "DT: mhpmcounter3_dcsr_stopcount.c, TODO: test hpmcounter[31-4]",
```

Comment:
What is this TODO?

Reply:
Aaa, yea, I have made the test mhpmcounter 3 specific, so I have not tested that stopcount works on counter 4 to 31. The todo is there to indicate that.